VAMobile/documentation/docs/Engineering/FrontEnd/DevSetupProcess.md

```diff
@@ -102,194 +83,173 @@ sidebar_position: 5
 
 ### Add file to proper directory
 
-   ![Google services file](/img/devSetupImage/google-service-android-file.png)
-
-- Open a terminal and type `nvm use` and press enter to set the node version for the project (if the version isn't installed, it will prompt you with the install command). If you don't have nvm you can install it from the [nvm documentation site](https://github.com/nvm-sh/nvm). Note: if you natively have the correct Node version installed you won't need nvm, but it's still good to have.
-
-- In a terminal type `yarn` or `yarn install` and press enter to install the projects dependencies. This will create the `node_modules` folder.
+![Google services file](/img/devSetupImage/google-service-android-file.png)
 
-- After dependencies are installed type `cd ios && pod install && cd ..` and press enter to install pods on ios (This is done once unless you installed new dependencies that need pods created).
+1. Open a terminal and type `nvm use` and press enter to set the node version for the project (if the version isn't installed, it will prompt you with the install command).
+2. In a terminal type `yarn` or `yarn install` and press enter to install the projects dependencies. This will create the `node_modules` folder.
```

Comment:
```suggestion
2. In a terminal, type `yarn` or `yarn install` and press enter to install the projects dependencies. This will create the `node_modules` folder.
```